recipes/openapi-generator/all/conanfile.py

```diff
@@ -20,7 +20,7 @@ def layout(self):
         pass
 
     def requirements(self):
-        self.requires("openjdk/19.0.2")
+        self.requires("openjdk/21.0.2")
```

Comment:
```suggestion
        self.requires("openjdk/21.0.1")
```

So it does not create a conflict with the other recipe that uses this major version, we can sync to a new patch verison in the future :)